NGitLab/Impl/HttpRequestor.cs

```diff
@@ -54,6 +55,11 @@ public virtual void Execute(string tailAPIUrl)
             Stream(tailAPIUrl, parser: null);
         }
 
+        public virtual async Task ExecuteAsync(string tailAPIUrl, CancellationToken cancellationToken)
+        {
+            await StreamAsync(tailAPIUrl, parser: null, cancellationToken).ConfigureAwait(false);
```

Comment:
Is the use of async/await necessary here? Or we can return the task itself?

Reply:
Oh my, it's going to be a big debate. I believe there are two schools.

 I agree with you I feel like it makes the code simpler to return the task directly.

Reply:
Fixed